### ISSUE TITLE
required ruby version >= 2.2

### ISSUE THREAD

**Issue by masutaka** [shunwen/fluent-plugin-rename-key#11]
fluent-plugin-rename-key requres ruby version >= 2.2 from f667339ef396da86386ae31b44bfd7da81bbc74e .

However latest td-agent (v2.3.3) has ruby-v2.1.10.
https://docs.treasuredata.com/articles/td-agent-changelog#20160930-version-v233

```
$ /opt/td-agent/embedded/bin/ruby --version
ruby 2.1.10p492 (2016-04-01 revision 54464) [x86_64-linux]
```

Is it the intended change?

Best regards.

**Comment by cosmo0920** [comment]
I think that it is NOT intended.
`required_ruby_version >= 2.1` is better.

Ideally, to support all of Fluentd v0.12 users, we should specify `required_ruby_version >= 1.9.3`.

**Comment by cosmo0920** [comment]
@shunwen How do you think about this?

**Comment by shunwen** [comment]
Sure, I dropped 1.9.3 because of this change: https://github.com/shunwen/fluent-plugin-rename-key/pull/10/commits/fcf74dff5b042a1f28713b35f677536f346edb43 . Changing it back to 1.9.3.

**Comment by shunwen** [comment]
@masutaka @cosmo0920 Fixed in 0.3.4 , thanks.

**Comment by masutaka** [comment]
I confirmed that it was resolved with v0.3.4. Thanks!
https://rubygems.org/gems/fluent-plugin-rename-key/versions/0.3.4
